autocrop/autocrop.py

```diff
@@ -128,11 +128,13 @@ def __init__(
         face_percent=50,
         padding=None,
         fix_gamma=True,
+        no_resize=False,
```

Comment:
Can we rename this attribute as `self.resize` in the `Cropper` class please? It's confusing to follow along with the double-negative 😄